README.md

```diff
@@ -114,6 +112,17 @@ your distribution:
     _Note: If the Graph Node you are deploying to requires authorization,
     make sure to authorize with the node using e.g. `graph auth http://127.0.0.`:8020 <ACCESS_TOKEN>`
     before deploying._
+    
+    You can also use the the npm script added into the `package.json`:
+    
+    ```sh
+    yarn deploy --verbosity debug
+    ```
+    
+    This will deploy the subgraph. If you want it to continuously upgrade, also pass the `watch` flag. If you are doing a lot of editing, and the subgraph is large, you will likely want to leave out the `watch` flag. 
```

Comment:
It's `--watch` flag, not `watch` flag.

Reply:
Also, drop the comment about large subgraphs. That doesn't make a lot of sense to me.